src/server/passport/strategies.js

```diff
@@ -0,0 +1,29 @@
+const strategyUtils = require('./strategyUtils')
+const UserModel = require('../models/user')
+
+const loginStrategy = (username, password, done) => {
+	const index = {
+		username
+	}
+	
+	const findUserHandler = strategyUtils.findUserHandler(username, password, done)
+	UserModel.findOne(index).exec()
```

Comment:
We can't stub the UserModel in test that way.

Reply:
ok